_data/publications.yml

```diff
@@ -1,3 +1,10 @@
+ - title: "Eyes on the Game: Deciphering Implicit Human Signals to Infer Human Proficiency, Trust, and Intent"
+   authors: Nikhil Hulle, Stéphane Aroca-Ouellette, Anthony J. Ries, Jake Brawer, Katharina von der Wense, Alessandro Roncone
+   year: 2024
+   venue: IEEE International Conference on Robot and Human Interactive Communication (ROMAN)
+   where: Pasadena, CA, USA, August 26-30
+   pdf: "2024_Nikhil-Stephane_RO-MAN_eyes_on_the_game"
+   subteams: [algorithmic hri and human-ai teaming, learning and modeling, stephane]
```

Comment:
please do not have `learning and modeling` here. (I know there is learning and modeling in this paper but it will be fixed soon) Thanks!

Reply:
Removed learning and modeling.